common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -572,6 +647,53 @@ innovation_chaosborn = {
 	flag = global_maa
 }
 
+innovation_draenei_tactics = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @peacekeeper
+
+	potential = {
+		this = culture:draenei
+	}
+
+	unlock_maa = peacekeeper
+	unlock_maa = rangari
+
+	flag = global_maa
+}
+
+innovation_demon_tactics = {
```

Comment:
Not existing unit type.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/155878891-f7af4ffe-2129-4a7d-8998-712e4d3b0159.png)

</details>